.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,64 @@
+name: integration-tests
+on:
+  push:
+    branches: [ "518-implement-semi-automatic-integration-tests" ]
+  pull_request:
+    branches: [ "518-implement-semi-automatic-integration-tests" ]
+
+env:
+  STORAGE_PATH: "/tmp"
+
+jobs:
+  deploy-and-test:
+    strategy:
+      matrix:
+        key: [8, 16]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checks-out repo
+        uses: actions/checkout@v2
+      - name: Install python
+        uses: actions/setup-python@v4
```

Comment:
Have you checked that python is not installed by default?

Reply:
fixed